src/Admin/Model/DoctrineRestServiceModel.php

```diff
@@ -968,13 +967,11 @@ public function updateContentNegotiationConfig(
      */
     public function updateDoctrineConfig(DoctrineRestServiceEntity $original, DoctrineRestServiceEntity $update)
     {
-        $patch                   = [];
+        $patch                   = ['zf-apigility' => ['doctrine-connected' => [$update->resourceClass => []]]];
         $patch['object_manager'] = $update->objectManager;
         $patch['hydrator']       = $update->hydratorName;
```

Comment:
This is the reason, it's going to add those two keys to global namespace (see #295)

maybe it would be better to write:

```php
<?php
public function updateDoctrineConfig(DoctrineRestServiceEntity $original, DoctrineRestServiceEntity $update)
{
    $patch = [
        'zf-apigility' => [
            'doctrine-connected' => [
                $update->resourceClass => [
                    'object_manager' => $update->objectManager,
                    'hydrator' => $update->hydratorName,
                ]
            ]
        ]
    ];

    $this->configResource->patch($patch);
}
```